test/test.jwt.ts

```diff
@@ -319,7 +286,6 @@ describe('JWT auth client', () => {
       jwt.request({uri: 'http://bar'}, () => {
         assert.equal('initial-access-token', jwt.credentials.access_token);
         assert.equal(false, scope.isDone());
-        nock.cleanAll();
```

Comment:
thanks for these

Reply:
👍  self preservation.  If I don't do it in a afterEach, tests fail in super confusing ways.